organization/external-communication.md

```diff
@@ -0,0 +1,168 @@
+
+### Promoting blog posts
+
+All blog posts should be promoted, regardless of whether or not they were written by a guest or a pyOpenSci employee! For promotion, coordinate with the Community Manager to ensure that there’s a tailored message for each of the following platforms:
+* Slack
+* Discourse
+* BlueSky
+* LinkedIn
+* Fosstodon
+
+In addition to pyOpenSci social outreach, we monitor social media for any personal posts that authors have created. To promote these posts, we will repost.
+
+### Blog post tone
+
+While we encourage guest authors to use a tone of voice that is authentic to them, the pyOpenSci blog uses a tone that is friendly, engaging, and curious. Although there are times when a post is written from the pyOpenSci perspective, and uses the terms "we" or even refers to pyOpenSci in the third person, it is more common for a post to be written from the author's perspective, and incorporate the use of "I."
+
+An example of this is the [pyOpenSci @ SciPy 2024 - Python Packaging Tutorials, Talks and Community post](https://www.pyopensci.org/blog/pyos-scipy-2024-recap.html), which incorporates personal anecdotes with data, images, and the author's reflections on the experience.
+
+## Newsletters
+
+The goals of the pyOpenSci newsletter are similar to our blog goals, with the addition of reaching a broader audience beyond website visitors. pyOpenSci uses LinkedIn as a newsletter platform, as many of our current (and future!) community members are on this platform. Using LinkedIn allows us to leverage our existing audience, and create consistent communication to help further community connections.
+
+We publish two newsletters:
+* A weekly newsletter on LinkedIn, published on Thursdays
+* A monthly newsletter published on LinkedIn and sent to all newsletter subscribers
+
+The weekly LinkedIn newsletter has three main categories:
+* The monthly round-up edition, sharing and celebrating pyOpenSci and community wins over the past month
+* A monthly "Community News" edition, which shares conversations, discussions, and decisions that have taken place in Slack as well as on Discourse and GitHub, that are relevant to the broader community
+* Re-posts of interesting pyOpenSci blog posts
+
+These posts are all structured as blog posts, using appropriate heading and subheading formats within LinkedIn. In fact, all of our newsletters are first published on the pyOpenSci blog, and should follow the same tone and format.
+
+When a newsletter post is a re-post from the pyOpenSci blog, it's important to include text and a link back to the original blog post, indicating where the post was originally published. This prevents link cannibalisation, where different links with the same content compete for keyword rankings. We want all keyword rankings to be directed back to the pyOpenSci website wherever possible.
```

Comment:
@kierisi don't we also want a canonical meta element in the html for this instance? So for eric's post we'd want to add a canonical tag to the html.

Reply:
that's a good point, I hadn't considered publishing other people's content on our website and will get this updated.